README.md

```diff
@@ -1,41 +1,85 @@
+FLAD Mentorship helps to link people to participate as a mentor or a mentee in the program [Let's talk about business](https://www.flad.pt/en/lets-talk-about-business/) from the Luso-American Development Foundation.
 
-In the project directory, you can run:
+## Development Notes
 
-### `npm start`
+FLAD-Mentorship app was bootstrapped with [Create React App](https://github.com/facebook/create-react-app). The project uses the Firebase Realtime Database and Storage as a backend, so it requires no server-side code or SQL queries.
 
-Runs the app in the development mode.<br>
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+## To build the application
 
-The page will reload if you make edits.<br>
-You will also see any lint errors in the console.
+The first step is to clone this repository in your local machine and then, open a terminal and cd into the project directory
 
-### `npm test`
+### 1. run `npm install`
 
-Launches the test runner in the interactive watch mode.<br>
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+This command will install all the necessary dependencies. [Material-ui](https://github.com/mui-org/material-ui), [firebase](https://github.com/firebase/), [react-to-print](https://www.npmjs.com/package/react-to-print) and [ramda](https://github.com/ramda/ramda) are some of them.
 
-### `npm run build`
+The file package.json contains the complete list of dependencies for this project.
 
-Builds the app for production to the `build` folder.<br>
-It correctly bundles React in production mode and optimizes the build for the best performance.
+### 2. Create a project in Firebase
+
+In order to try this app you may create a new project in the Firebase Console. You can create a free account (here) [https://console.firebase.google.com/]
+
+### 3. Enable Email/Password as authentication method in your Firebase project
+
+This step is important in order to be able to create accounts in your Firebase project and try the functionality available for the different roles of the program: mentor, mentee and coordinator.
+
+To enable Email/Password as authentication method in your Firebase project:
+
+- Go to the [Firebase console](https://console.firebase.google.com/), open the Auth section.
+- On the Sign in method tab, enable the Email/password sign-in method and click Save.
+
+### 4. Create a folder for images
+
+Next you will need to create a folder to storage the pictures of the mentors participating in FLAD-Mentorship program.
+
+- Go to the [Firebase console](https://console.firebase.google.com/), open the Storage section.
+- On the Files tab, click in the icon "create folder".
+- Name your new folder **images**
+
+### 5. Set the environment variables
+
+Once you have created your project in Firebase, create manually the file .env.production in the root of your project. Type the keys generated to connect to your database as follow:
 
```

Comment:
Shall we highlight the files's name? `.env.production`
Do we need the `.env.dev` or just the production one?

Reply:
I think collaborators will need just .env.production, but as a good practice, we can recommend both. I made the changes. 😉